src/apexguru/apex-guru-service.ts

```diff
@@ -57,7 +57,9 @@ export async function runApexGuruOnFile(selection: vscode.Uri, runInfo: RunInfo)
 			new DiagnosticManager().displayDiagnostics([selection.fsPath], [ruleResult], diagnosticCollection);
 			TelemetryService.sendCommandEvent(Constants.TELEM_SUCCESSFUL_APEX_GURU_FILE_ANALYSIS, {
 				executedCommand: commandName,
-				duration: (Date.now() - startTime).toString()
+				duration: (Date.now() - startTime).toString(),
+				violationsCount: ruleResult.violations.length.toString(),
+				violationsWithSuggestedCode: getViolationsWithSuggestions(ruleResult).toString()
```

Comment:
Should this be `violationsWithSuggestedCodeCount` to match `violationsCount`?

Reply:
Yes! that sounds better. I'll rename it.